stdlib/Dates/test/types.jl

```diff
@@ -228,6 +228,20 @@ end
     @test Dates.Date(Dates.DateTime(Dates.Date(2012, 7, 1))) == Dates.Date(2012, 7, 1)
 end
 
+@testset "min and max" begin
+    for (a, b) in [(Dates.Date(2000), Dates.Date(2001)),
+                    (Dates.Time(10), Dates.Time(11)),
+                    (Dates.DateTime(3000), Dates.DateTime(3001)),
+                    (Dates.Week(42), Dates.Week(1972))]
+        @test min(a, b) == a
+        @test min(a)    == a
+        @test max(a, b) == b
+        @test max(b)    == b
+        @test minmax(a, b) == (a, b)
+        @test minmax(a)    == (a, a)
+    end
+end
```

Comment:
You might want to test these in the other direction as well, so `min(b, a) == a`, etc.

Reply:
good suggestion - done